### PR TITLE
add FAQ about themes

### PR DESCRIPTION
**Pull request by wweich**
Add an FAQ entry about how the change the theme, add custom theme and add custom GUI.

### CODE REVIEW

File: users/faq.rst [hunk 1]
```diff
@@ -315,6 +315,29 @@ protect against unauthorized access. Either:
 
 In all cases, username/password authentication and HTTPS should be used.
 
+I don't like the GUI / Theme. Can it be changed?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can change the theme in the settings. Syncthing ships with a default and
```
Comment (canton7) [inline users/faq.rst]: > As of 0.14.10, there is also a black theme, which is the old dark theme with black background.

I think this is just a bit confusing, and will be useless in a couple of month's time. Why not say it ships with other themes, and leave it at that?

File: users/faq.rst [hunk 2]
```diff
@@ -315,6 +315,29 @@ protect against unauthorized access. Either:
...
+a dark theme. As of 0.14.10, there is also a black theme, which is the old
+dark theme with black background.
+
+If you want a custom theme or a completly different GUI, you can add your
+own.
+By default, Syncthing will look for a directory ``gui`` inside the Syncthing
+home folder. To change the directory to look for themes, you neeed to set the
+STGUIASSETS environment variable. To get the concrete diroctory, run
```
Comment (canton7) [inline users/faq.rst]: "diroctory"

File: users/faq.rst [hunk 3]
```diff
@@ -315,6 +315,29 @@ protect against unauthorized access. Either:
...
+syncthing with the ``-paths`` paremeter. It will print all the relevent paths,
+including the "GUI override directory".
+
+To add e.g. a red theme, you can just create the directory
+``red/assets/css/`` and the file ``theme.css`` inside the new css folder to override the
```
Comment (canton7) [inline users/faq.rst]: "you can create the file ``red/assets/css/theme.css`` inside the GUI override directory?

File: users/faq.rst [hunk 4]
```diff
@@ -315,6 +315,29 @@ protect against unauthorized access. Either:
...
+default CSS styles.
+
+To create a whole new GUI, you should checkout the files at
+https://github.com/syncthing/syncthing/tree/master/gui/default
+to get an idead how to do that.
```
Comment (canton7) [inline users/faq.rst]: "idead"

**Comment by canton7** [comment]
@st-review lgtm

**Comment by st-review** [comment]
@canton7: Noted! Need another LGTM or explicit merge command.

File: users/faq.rst [hunk 5]
```diff
@@ -315,6 +315,28 @@ protect against unauthorized access. Either:
...
+
+You can change the theme in the settings. Syncthing ships with other themes
+than the default.
+
+If you want a custom theme or a completly different GUI, you can add your
+own.
+By default, Syncthing will look for a directory ``gui`` inside the Syncthing
+home folder. To change the directory to look for themes, you neeed to set the
+STGUIASSETS environment variable. To get the concrete directory, run
+syncthing with the ``-paths`` paremeter. It will print all the relevent paths,
```
Comment (canton7) [inline users/faq.rst]: "paremeter"

**Comment by canton7** [comment]
Oops, I take back that lgtm...

**Comment by AudriusButkevicius** [comment]
@st-review lgtm

**Comment by st-review** [comment]
:ok_hand: Merged as 078da88323ed0be664e201f870553e686a31519c. Thanks, @wweich!

**Pull request closed without merging**